knotx-it-tests/src/test/java/io/knotx/example/monolith/SampleApplicationIntegrationTest.java

```diff
@@ -176,6 +178,20 @@ public void whenRequestingLocalSimplePageWithCustomTag_expectLocalSimpleHtml(
         "localSimpleResultAngular.html");
   }
 
+  @Test
+  @KnotxApplyConfiguration("knotx-test-app.conf")
+  public void whenRequestingFailingServiceWithFallback_expectFallback(
+      VertxTestContext context, Vertx vertx) {
+    testGetRequest(context, vertx, LOCAL_FALLBACK_REQUEST_URI, "localFallbackResult.html");
+  }
+
+  @Test
+  @KnotxApplyConfiguration("knotx-test-app-global-fallback.conf")
```

Comment:
You may add 
```hocon
    defaultFallback = MY_GLOBAL_ID
    fallbacks = [
      {
        id = MY_GLOBAL_ID
        markup = "<p class='error'>global fallback</p>"
      }
    ]
```
directly to the `knotx-test-app.conf`.
If you want to add some extra config, please don't copy the whole file (if we do the change in the future we will have to change in multiple places). You may override just a part of a config, `@KnotxApplyConfiguration` supports this.
Look at the `whenRequestingLocalSimplePageWithGetCustomAndDefaultSymbol_expectLocalSimpleHtmlWithDefault`. It defines two config files:
`@KnotxApplyConfiguration({"knotx-test-app.conf", "knotx-test-handlebars-custom-symbol.conf"})`
Where `knotx-test-handlebars-custom-symbol.conf` overrides some configs from the `knotx-test-app.conf`.